<!--
  This file is part of PanDrugs Backend.

  PanDrugs Backend is free software: you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation, either version 3 of the License, or
  (at your option) any later version.

  PanDrugs Backend is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.

  You should have received a copy of the GNU General Public License
  along with PanDrugs Backend.  If not, see <http://www.gnu.org/licenses/>.
-->

<div class="container-fluid">
  <div class="row" id="intro">
    <div class="col-lg-6">
      <div class="jumbotron">

        <h1>Welcome to</h1>
        <p class="lead">
          <img id="logo-big" src="images/pandrugs2-logo.svg" class="img-responsive"><br>
          A novel method for prioritizing therapies using individual multi-omics data
        </p>
        <p><a class="btn btn-lg btn-success" ng-href="#!/query">Query!
            <span class="glyphicon glyphicon-ok"></span></a></p>

      </div>
    </div>
    <div class="col-lg-6" ng-init="hometab = 'workflow'">
      <h3>What is PanDrugs?</h3>
      <p class="db_desc"><strong>PanDrugs</strong> provides a bioinformatics platform
        <strong>to prioritize anticancer drug treatments according to individual multi-omics
          data</strong>. The current version of PanDrugs database integrates data from <strong>23</strong> primary
        sources and stores <strong>74,087</strong> drug-target associations obtained
        from <strong>4,642</strong> genes and <strong>14,659</strong> unique compounds.
      </p>

      <p class="db_desc">
        <strong>Data input</strong>:
      <ol type="a">
        <li><strong>VCF file</strong>: for somatic and germline variant analysis.</li>
        <li><strong>RNK file</strong>: for pre-ranked gene lists.</li>
        <li><strong>TSV file</strong>: for CNV analysis.</li>
        <li><strong>Gene lists</strong>.</li>
        <li><strong>Drug queries</strong>.</li>
      </ol>
      </p>
      <p class="db_desc">Please note PanDrugs terminology for druggable genes:
      <ol type="I">
        <li><strong>Direct targets</strong>: Genes that contribute to a disease phenotype and can be directly
          targeted by a drug (e.g. <i>BRAF</i> is a direct target of vemurafenib).</li>
        <li><strong>Biomarkers</strong>: Genes whose genetic status is associated with a drug response
          but are not the drug target themselves (e.g. <i>BRCA</i>-mutated
          cancers responding to PARP inhibitors).</li>
        <li><strong>Genetic dependencies</strong>: Druggable genes upon which an altered
          gene fuctionally depends (e.g. patients with <i>BRAF</i> GoF treated with BRAF and MEK inhibitors
          can develop resistance involving MAPK signaling reactivation. Thus, PanDrugs suggests targeting
          <i>MAP2K1</i> in tumors with <i>BRAF</i> GoF).
        </li>
        <li><strong>Pathway members</strong>: Druggable genes found within the pathway and downstream
          of an altered gene (e.g. patients with mutations in <i>TSC1/2</i>
          respond to downstream inhibition of the mTOR pathway).</li>
      </ol>
      </p>
      <img src="images/interactions.svg" id="interactions-legend">
    </div>
  </div>
  <div id="method">
    <h3>PanDrugs workflow</h3>
    <p>PanDrugs considers drug indication and approval status, the number of genes associated
      with a drug, the nature and reliability of these associations and the type of drug response to
      calculate the <strong>Drug Score (DScore)</strong>. <strong>The Gene Score (GScore)</strong> is
      estimated according to gene essentiality, tumour vulnerability, the relevance of the gene
      in cancer, its druggability level, the biological impact of mutations, the frequency of
      gene alterations and their clinical implications. PanDrugs resolves the
      <strong>Best Therapeutic Candidates (BTC)</strong> based on both the <strong>DScore</strong>
      and the <strong>GScore</strong>.
    </p>
    <img src="images/pandrugs-scoring.svg">
  </div>
  <div class="alert alert-warning">
    <strong>Disclaimer</strong>: this resource is exclusively intended <strong>for research purposes
      and academic use</strong>. It should not be used for medical or professional advice.
  </div>
  <div class="alert alert-info">
    <strong>Please cite us</strong>:
    <div>Jiménez-Santos, M. J. <i>et al.</i> (2023).
      <a href="https://doi.org/10.1093/nar/gkad412" target="_blank" style="color:#000000">
        PanDrugs2: prioritizing cancer therapies using integrated individual multi-omics data.</a> <i>Nucleic Acids Research. 51</i>(Web Server Issue W1), pp. W411-W418.
    </div>
    <div>Piñeiro-Yáñez, E. <i>et al.</i> (2018).
      <a href="https://doi.org/10.1186/s13073-018-0546-1" target="_blank" style="color:#000000">
        PanDrugs: a novel method to prioritize anticancer drug treatments according to individual
        genomic data.</a> <i>Genome medicine, 10</i>(1), 41.
    </div>

  </div>
</div><!-- end of the container -->

<div id="datapolicy" class="modal fade" tabindex="-1" role="dialog">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>

        <h4 class="modal-title">PanDrugs data policy</h4>
      </div>


    </div>
  </div>
</div>